src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  max-width: calc(7 * (#{$cell-size} + #{$border-width} + #{$gap} + #{$padding}));
+
+  &--start-day-mon {
+    .calendar__cell:nth-child(1) {
+      margin-right: calc((6 * #{$cell-size}) + (6 * #{$border-width}) + (6 * #{$gap}) + (2 * #{$padding}));
+      width: #{$cell-size};
+      height: #{$cell-size};
+    }
+  }
+
+  @for $dayWeek from 28 through 31 {
+    $monthLength: $dayWeek;
+
+    &--month-length-#{$monthLength} {
+      @include generate-calendar-cells($monthLength);
+
+      .calendar__day:nth-child(n + #{$monthLength + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  $start-days: mon, tue, wed, thu, fri, sat, sun;
```

and move it to variable file
```suggestion
  $start-days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
```